interface/src/app/node/high_range_exp.tpl.html

```diff
@@ -0,0 +1,48 @@
+<div ng-if="!queryStatus">
+  <h4>
+    <b>Experiments with the highest range: </b>{{experiments.length}}
+  </h4>
+  <button class="btn btn-primary" ng-click="showTop()"
+          ng-hide="topMode">Show Top {{topNum}}</button>
+  <button class="btn btn-primary" ng-click="showAll()"
+          ng-show="topMode">Show All</button>
+
+  <br />
+  (Click
+  <i class="fa fa-plus-square fa-lg" aria-hidden="true"></i> or
```

Comment:
I just took a look. I worry that there are too many "click here" type indications with the full button. For a user it can be confusing to have lots of branching paths with unclear priority.